src/components/dialogs/load-modification-dialog.js

```diff
@@ -183,8 +171,8 @@ const LoadModificationDialog = ({
                 loadType,
                 activePower,
                 reactivePower,
-                connectivity?.voltageLevel?.id,
-                connectivity?.busOrBusbarSection?.id,
+                undefined,
```

Comment:
Is `undefined` value overwrites the Load connectivity member backward ?
Not sure it is ignored.

Reply:
tested, connectivity was already always undefined since the fields were disabled, it does not change the API calls